app/views/compute_resources_vms/form/vmware/_network.html.erb

```diff
@@ -1,12 +1,12 @@
 <%= select_f f, :type, compute_resource.nictypes, :first, :last, { },
   :class => "col-md-3 vmware_type",
-  :label => _('NIC type'), :label_size => "col-md-3", :disabled => !new_host
+  :label => _('NIC type'), :label_size => "col-md-3", :disabled => !new_vm
 %>
-<% if new_host %>
+<% if new_vm %>
   <%= select_f f, :network, vsphere_networks(compute_resource), :first, :last, { },
     :class => "col-md-3 vmware_network",
     :label => _('Network'), :label_size => "col-md-3"
   %>
 <% else %>
-  <%= text_f f, :network, :class => 'col-md-3 vmware_network', :label => _("Network"), :label_size => "col-md-3", :disabled => true %>
+  <%= text_f f, :network, :class => 'col-md-3 vmware_network', :label => _("Network"), :label_size => "col-md-3", :readonly => true %>
```

Comment:
When the field is disabled, the value doesn't get serialized and when the host is saved it doesn't save the `interfaces_attributes`.